app/pillar/test/pillar_test.go

```diff
@@ -6,11 +6,51 @@ import (
 	"fmt"
 	"github.com/coralproject/pillar/pkg/db"
 	"github.com/coralproject/pillar/pkg/model"
+	"github.com/coralproject/pillar/pkg/web"
+	"log"
 	"os"
 	"testing"
 )
 
+//various constants
+const (
+	MethodGet    string = "GET"
+	MethodPost   string = "POST"
+	MethodOption string = "OPTIONS"
+
+	URLUser    string = "api/import/user"
+	URLAsset   string = "api/import/asset"
+	URLAction  string = "api/import/action"
+	URLComment string = "api/import/comment"
+	URLTag     string = "api/tag"
+	URLTags    string = "api/tags"
+
+	DataUsers    = "fixtures/users.json"
+	DataAssets   = "fixtures/assets.json"
+	DataComments = "fixtures/comments.json"
+	DataActions  = "fixtures/actions.json"
+	DataTags     = "fixtures/tags.json"
+)
+
+var baseURL string
+
+func getBaseURL() string {
+	return baseURL
+}
+
+func getHeader() {
+	m := make(map[string]string)
+	m["Content-Type"] = "application/json"
+
+	return m
+}
+
 func init() {
```

Comment:
Be careful with the init in tests. I use a setup() function (and teardown() for cleaning db if needed) and then add a TestMain(m *testing.M) that defines how to run the tests.